README.rst

```diff
@@ -22,7 +22,7 @@ This package is compatible and tested with latest versions of Django (1.8, 1.9,
 Install
 =======
 
-::
+.. code-block:: bashus
```

Comment:
Can you provide a reference of what `bashus` code block is for?

Reply:
@romgar it's a typo, see #144